docs/running-notebooks/listing-active-sessions/index.md

```diff
@@ -0,0 +1,13 @@
+# Listing active sessions
+
+You can list the active Jupyter notebooks sessions from the REANA-Client
+the same way you list workflows and additionally passing the `-s/--sessions` flag:
+
+```console
+$ reana-client list --sessions
+NAME         RUN_NUMBER   CREATED               SESSION_TYPE   SESSION_URI                                                                           SESSION_STATUS
+jupyter      1            2021-12-01T14:33:26   jupyter        https://reana.cern.ch/f8be55e4-5d18-43f9-b977-f773fdcab163?token=<your-reana-token>   created
+```
+
+There you have information about the session workflow, session URI and
```

Comment:
Minor: `There you have information about the session workflow,` -> `There you can find information about the session workflow name,..`, WDYT?